jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -159,7 +160,7 @@ def user_api(self):
         expose = ['function', 'spatial_subset', 'aperture',
                   'add_results', 'collapse_to_spectrum',
                   'wavelength_dependent', 'reference_wavelength',
-                  'aperture_method']
+                  'aperture_method', 'translate_units']
```

Comment:
I'd vote to not expose this to the user yet.

Reply:
The list name makes more sense now, will do and make a note of it.

Reply:
@kecnry how do I hide it from the use, but still enable it to be callable?

Reply:
remove it from this list, and either access the plugin object directly (if you already have the user API object, just call `._obj` on it to get to the actual plugin object).